src/data/markdown/docs/30 xk6-browser/01 xk6-browser/09 Locator/inputvalue--options--.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'inputValue([options])'
+excerpt: 'xk6-browser: locator.inputValue method'
+---
+
+Returns `input.value` for the selected `input`, `textarea` or `select` element.
+
+| Parameter | Type   | Description                               |
+| --------- | ------ | ----------------------------------------- |
+| options   | object | See [options](#options) for more details. |
+
+### Returns
+
+| Type   | Description                      |
+|--------|----------------------------------|
+| string | The input value of the selector. |
```

Comment:
```suggestion
| string | The input value of the element. |
```

Reply:
Resolved in: https://github.com/grafana/k6-docs/commit/22dad9ff4b1a8166a764a18b389fca18538b9449